### PR TITLE
Adding the missing Legos from the runbooks

### PR DESCRIPTION
**Pull request by jayasimha-raghavan-unskript**
## Description
Adding the missing legos from the Runbooks.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: MySQL/legos/mysql_read_query/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>Long Running MySQL Query</h1>
+
+## Description
+This Lego queries the database and retrieves long run queries.
```
Comment (abhishek-unskript) [inline MySQL/legos/mysql_read_query/README.md:5]: this is not the correct description I think

File: MySQL/legos/mysql_read_query/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## Lego Details
+
+    mysql_read_query(handle: object)
+
+        handle: Object of type unSkript MySQL Connector
+
+## Lego Input
+This Lego takes twoinput query and param.
```
Comment (abhishek-unskript) [inline MySQL/legos/mysql_read_query/README.md:15]: Parameters should be shown in detail with examples?

File: MySQL/legos/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+    <h2>unSkript Legos </h2>
+
+
+# MySQL Lego Collection
+* [Run MySQL Query](./mysql_read_query/README.md)
```
Comment (abhishek-unskript) [inline MySQL/legos/README.md:6]: There are two legos underneath, not one

File: MySQL/legos/mysql_kill_long_query/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>Kill Long Running MySQL Query </h1>
+
+## Description
+This Lego queries the database and retrieves long run queries.
```
Comment (abhishek-unskript) [inline MySQL/legos/mysql_kill_long_query/README.md:5]: "retrieves long running queries"

File: MySQL/legos/mysql_kill_long_query/mysql_read_query.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+    "action_title": "Kill Long Running MySQL Query",
+    "action_description": "Kill Long Running MySQL Query",
+    "action_type": "LEGO_TYPE_MYSQL",
+    "action_entry_function": "mysql_read_query",
+    "action_needs_credential": true,
+    "action_output_type": "ACTION_OUTPUT_TYPE_LIST",
+    "action_supports_poll": true,
+    "action_supports_iteration": true
+  }
```
Comment (abhishek-unskript) [inline MySQL/legos/mysql_kill_long_query/mysql_read_query.json:10]: File name and entry function name are not correct.

File: MySQL/legos/mysql_kill_long_query/mysql_read_query.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+class InputSchema(BaseModel):
+    query: str = Field(
+        title='Read Query',
+        description='MySQL get query.')
+    params: List = Field(
+        None,
+        title='Parameters',
+        description='Parameters to the query in list format. For eg: [1, 2, "abc"]')
+
```
Comment (abhishek-unskript) [inline MySQL/legos/mysql_kill_long_query/mysql_read_query.py:19]: I don't think this is the correct lego. Perhaps you have copied the wrong files. Please relook thru this diff.

**Review by abhishek-unskript — CHANGES_REQUESTED** [review]

**Review by twishek — APPROVED** [review]

**Review by abhishek-unskript — APPROVED** [review]

**Pull request merged**